Bottom: 7ad2343bd49f466434bdd06d38671bfc4fab9c23
Top:    b9ba4dcdb960d5ea5aeca617392289011f30e9bb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-11 16:44:06 +0000

Refresh of temp-fixes-and-diagnostics

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 74a01bc..2d65872 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1162,7 +1162,7 @@ label_contour (PLINE * a)
 	}
     }
   while ((cur = cur->next) != first_not_unknown &&
-         !(cur->next == &a->head && first_not_unknown == NULL));
+	 !(cur->next == &a->head && first_not_unknown == NULL));
 
   if (first_not_unknown == NULL)
     fprintf (stderr, "OOPS, DIDN'T LABEL ANYTHING!\n");
